test/test_cli.py

```diff
@@ -215,3 +218,21 @@ def test_print_format(monkeypatch, vuln_count, pkg_count, skip_count, print_form
         pass
 
     assert bool(dummyformat.format.calls) == print_format
+
+
+def test_environment_variable():
+    """Environment variables set before execution change CLI option default."""
+    with ArgvContext("pip-audit"), EnvironContext(
+        PIP_AUDIT_DESC="off",
+        PIP_AUDIT_FORMAT="markdown",
+        PIP_AUDIT_OUTPUT="stderr",
+        PIP_AUDIT_PROGRESS_SPINNER="off",
+        PIP_AUDIT_VULNERABILITY_SERVICE="osv",
+    ):
+        args = pip_audit._cli._parser().parse_args()
```

Comment:
It'll be a little less clean, but my preference here would be to use `pytest`'s built-in monkeypatching (via the `monkeypatch` fixture) for argv/environment patching here, rather than adding an additional test-only dependency.